tests/testutil/testutil.go

```diff
@@ -132,11 +133,24 @@ func stringInSlice(a string, list []string) bool {
 // KUBECONFIG env value
 func GetKubeClient() *kubernetes.Clientset {
 	kubeConfigENV := os.Getenv("KUBECONFIG")
+	if kubeConfigENV == "" {
+		if home := homeDir(); home != "" {
+			kubeConfigENV= filepath.Join(home, ".kube", "config")
+		}
```

Comment:
else maybe throw a descriptive error here to let the tester know to set the KUBECONFIG environment variable

Reply:
I also thought on exiting with a terminal error code (Exit 1) if this happens.